src/Gateway.php

```diff
@@ -54,6 +54,20 @@ public function purchase(array $parameters = array())
         return $this->createRequest('\Omnipay\Stripe\Message\PurchaseRequest', $parameters);
     }
 
+    /**
+     * Update charge request.
+     *
+     * Update the data for a charge after the charge has been created.
+     *
+     * @param array $parameters
+     *
+     * @return \Omnipay\Stripe\Message\Response
```

Comment:
Return type should be
`     * @return \Omnipay\Stripe\Message\UpdateChargeRequest
`